addons/mail/models/mail_message.py

```diff
@@ -778,8 +779,9 @@ def toggle_message_starred(self):
             self.sudo().write({'starred_partner_ids': [Command.unlink(self.env.user.partner_id.id)]})
 
         self.env['bus.bus']._sendone(self.env.user.partner_id, 'mail.message/toggle_star', {
-            'message_ids': [self.id],
             'starred': starred,
+            'starred_counter': self.env.user.partner_id._get_starred_count(),
```

Comment:
Now that I think about it, it seems way too expensive to have to go through all the messages every time a message is (un)starred. Would it be possible to fix the issue addressed in this PR without having to recount the starred messages?